test.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import json
+
+class NpEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, np.integer):
+            return int(obj)
+        elif isinstance(obj, np.floating):
+            return float(obj)
+        elif isinstance(obj, np.ndarray):
+            return obj.tolist()
+        else:
+            return super(NpEncoder, self).default(obj)
+
+# netpyne/sim/save.py#L37
+#json.dumps(data, cls=NpEncoder)
+
+a = 1
+b = 1.0 
+c = np.array(3)
+
+
+string = json.dumps(c,indent=4, sort_keys=True,
+                    separators=(',', ': '), ensure_ascii=False, cls=NpEncoder)
+print(string)
```

Comment:
This would require a person looking at the result. We need to compare the response to something we know.

```python
assert string == "something I know"
```

For now, let's leave the testing outside the scope of this PR. Testing requires the definition of a framework (pytest, unitest) and the inclusion in `.travis.yml`.